package.json

```diff
@@ -24,9 +24,9 @@
     "protoc": "bash ./scripts/update_proto.sh"
   },
   "dependencies": {
-    "@dfinity/agent": "^0.11.1",
-    "@dfinity/candid": "^0.11.1",
-    "@dfinity/principal": "^0.11.1",
+    "@dfinity/agent": "^0.11.2",
```

Comment:
as I guess it is not related, can you please not update agent in this PR?

Reply:
Done.